From e7514f21e76fba973360ff4da24b77aae5272b21 Mon Sep 17 00:00:00 2001
From: Luis Chamberlain <mcgrof@kernel.org>
Date: Thu, 12 May 2022 11:55:29 -0700
Subject: [PATCH 20/20] MAINTAINERS: add Russ Weight as a firmware loader
 maintainer

Russ has done extensive rework on the usermode helper interface for
the firmware loader. He's also exressed recent interest with maintenance
and has kindly agreed to help review generic patches for the firmware
loader. So add him as a new maintainer!

Signed-off-by: Luis Chamberlain <mcgrof@kernel.org>
Signed-off-by: Russ Weight <russell.h.weight@intel.com>
Link: https://lore.kernel.org/r/20220512185529.3138310-1-mcgrof@kernel.org
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index d6d879cb0..91e72f232 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7656,6 +7656,7 @@ F:	include/linux/arm_ffa.h
 
 FIRMWARE LOADER (request_firmware)
 M:	Luis Chamberlain <mcgrof@kernel.org>
+M:	Russ Weight <russell.h.weight@intel.com>
 L:	linux-kernel@vger.kernel.org
 S:	Maintained
 F:	Documentation/firmware_class/
-- 
2.36.1.74.g277cf0bc36

